### PR TITLE
Update Bundled App Installation Instructions

### LINKED ISSUE #67: Updating napari's Bundled App Installation Instructions
This is a follow up to napari/napari#4383. Please see the discussion there.

### LINKED ISSUE #288: Clarify napari bundled app installation to ensure stable release and not RC
## 🧰 Task
The bundled app instructions here:
https://napari.org/stable/tutorials/fundamentals/installation.html#install-as-a-bundled-app
image
That first link goes to the Releases GitHub page which currently has RC 0.4.19 at the top, which could lead to confusion, see:
https://forum.image.sc/t/why-napari-bundled-app-cannot-install-the-plugin-napari-allencell-segmenter/89419/4
The 2nd link does point to the correct version, but I think we should shuffle this and add a note/warning that older and experimental versions are also available.

Related: https://github.com/napari/docs/issues/67

### PR DESCRIPTION
**Pull request by jaimergp**
# Description

Add detailed installation instructions for the new conda-based bundled app, updated with screenshots from 0.5.0+.

# References

Closes #67

This PR brings napari/napari#4383 over to `napari/docs`. It's not a perfect port, and I had to edit some patches, but it keeps most of the original history.

Inspired by https://github.com/napari/napari/pull/5240#issuecomment-1282076004, I followed these steps:

1. Obtain the commit-by-commit patch from the original PR:

```bash
$ cd ~/devel/napari/napari
$ gh pr checkout 4383
$ gh pr diff --patch > ~/tmp/napari-4383.patch
```

2. Apply patch interactively on `napari/docs`:

```bash
$ cd ~/devel/napari/docs
# Create new branch from up-to-date main
$ git checkout main
$ git pull upstream main
$ git checkout -b bring-napari-4383
# Apply patch
$ git am ~/tmp/napari-4383.patch
```

Sadly, the naive `git am` command with no flags didn't work, so I had to do it "interactively" with `-i --rejected`, which iteratively creates `*.rej` files with the rejected chunks of each patch:

```bash
$ git am -i --rejected ~/tmp/napari-4383.patch
# analyze rejection and modify target file; then:
$ git am -i --rejected --continue
# rinse and repeat til you are done
```

---

### CODE REVIEW

**Comment by jaimergp** [comment]
Loving the `<kbd>` default style!

image

**Comment by psobolewskiPhD** [comment]
Gentle reminder about this one @jaimergp Pretty high-value contribution!!
❤️

**Comment by jaimergp** [comment]
I don't remember what was blocking this, but I think I recall it was something about conveying the right kind of messaging in the website. Are we ok to publish this now? @psobolewskiPhD 

Assuming we are, I think it would just need to update the screenshots so it shows 0.5.0 instead of 0.4.18-ish, and maybe polish some text here and there.

Are we still pushing images directly in the git repo or is there another mechanism I should be aware of?

**Comment by psobolewskiPhD** [comment]
@jaimergp I think the bundle was considered experimental? but now it's pretty stable, right?
Anyhow, I think it's definitely something we want in the docs.
Nothing has changed with images, just commit and push.

**Comment by melissawm** [comment]
Hi all! Is there anything I can do to help push this forward? Cheers! 😄

**Comment by jaimergp** [comment]
Now that we have 0.5.x out, we can take the screenshots and update the text.

**Comment by jaimergp** [comment]
@melissawm - do you happen to have a Linux (VM or not) machine for x64? I can only virtualize ARM 😬

**Comment by melissawm** [comment]
@jaimergp yes I do, happy to help!

**Comment by jaimergp** [comment]
Excellent, we'd just need to update the screenshots `docs/tutorials/assets/tutorials/installation/bundle_{28..35}.png` with the most recent Linux SH installer at https://github.com/napari/napari/releases/tag/v0.5.0.

File: docs/tutorials/fundamentals/installation_bundle_conda.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+You can now get started using napari! A shortcut to launch napari can be found in the Windows Start menu. 
+
+napari comes installed with sample images from scikit-image. Use the dropdown menu File > Open Sample > napari to open a sample image, or open one of your own images using File > Open or dragging and dropping your image onto the canvas. 
+
+Next check out our [tutorial on the viewer](viewer.md) or explore any of the pages under the [Usage tab](../../usage.md).
+
+### How to Install the Linux bundle
+
+Once you have downloaded the Linux SH file, you will have a file with a name like `napari-0.4.15-Linux-x86_64.sh`. You can double-click this file to open the installer in a terminal, or, alternatively, open a terminal yourself and run the command `bash napari-*-Linux-x86_64.sh`.
```
Comment (melissawm) [inline docs/tutorials/fundamentals/installation_bundle_conda.md]: nitpick, but double clicking is not consistent across Linux distros/WM so I'd not even mention that.

**Review by melissawm — COMMENTED** [review]
Instructions work fine on Manjaro - do you need anything else? Updated screenshots of the viewer? For the terminal screenshots, I think the Ubuntu ones are more common tbh.

EDIT: Got it, will push to this PR in a few mins.

**Comment by jaimergp** [comment]
> Updated screenshots of the viewer?

It's mostly about the version string in the screenshots. Having 0.5.0 looks cleaner than a dev version of 0.4.17 😬

I need to update the text too. I'll take a look at your suggestion, thanks!

**Comment by melissawm** [comment]
Let me know if you need something different!

**Comment by jaimergp** [comment]
Looks amazing, thanks @melissawm!

**Review by psobolewskiPhD — DISMISSED** [review]
This looks super!
Could we get it better integrated with the installation guide?
https://output.circle-artifacts.com/output/job/a8a5e740-214a-47ab-9c56-fed5228432a4/artifacts/0/docs/docs/_build/html/tutorials/fundamentals/installation.html#install-as-a-bundled-app
I think this should be all but dropped and just replaced with a link to the new guide?

**Comment by jni** [comment]
@jaimergp we just discussed this at community meeting. At a minimum before merge we would like to add a link to this new guide [in this page](https://napari.org/dev/tutorials/fundamentals/installation.html#install-as-a-bundled-app) ([source link](https://github.com/napari/docs/blob/be5bf6c06c5f515c08f16261fb4e3683ea7aea1d/docs/tutorials/fundamentals/installation.md?plain=1#L230)). We are planning to release later today, so if you have a chance to add a small link 🙏 that would be great, otherwise, let us know and we can push directly.

**Comment by jaimergp** [comment]
How does this look: https://output.circle-artifacts.com/output/job/fb66cbb2-7b72-4b9e-9aa2-4199b5844109/artifacts/0/docs/docs/_build/html/tutorials/fundamentals/installation.html#install-as-a-bundled-app

**Comment by jni** [comment]
it looks good to me, thanks @jaimergp — I'm just gonna give @psobolewskiPhD a chance to have another look at it, but will just merge in the morning if he doesn't get a chance.

**Review by jni — APPROVED** [review]

**Comment by psobolewskiPhD** [comment]
Sorry I dropped the ball on this -- long day 😞

**Comment by jni** [comment]
all is well @psobolewskiPhD, I just hope I didn't offend you with

Screenshot 2024-07-25 at 1 02 20 PM

😂 🙏

**Pull request merged**